apps/search/api/filters/pre_resolve_filters.py

```diff
@@ -0,0 +1,31 @@
+
+SORT_BY_PARAM = 'ordering'
+
+# list of filters that's applied to every module
+PRE_RESOLVE_FILTERS = [
+    # random ordering filter could be applied as a hack to disable caching search queries via cachalot
```

Comment:
@alashow, do you think it's actually a problem that cachalot caches search queries? Also, can you explain what is meant by "random queries"?

Reply:
Depends on server memory + search usage. We're not actually gaining anything from caching search results because they're already fast enough. It will only be a problem when there's a lot of search queries.